src/rtextures.c

```diff
@@ -275,6 +282,88 @@ Image LoadImageRaw(const char *fileName, int width, int height, int format, int
     return image;
 }
 
+// Load image from SVG file data with default size from SVG
+Image LoadImageSvg(const char* fileName)
+{
+    Image image = { 0 };
+
+    unsigned int dataSize = 0;
+    unsigned char* fileData = LoadFileData(fileName, &dataSize);
+
+    if (fileData != NULL)
+    {
+    	struct NSVGimage* svgImage = nsvgParse(fileData, "px", 96.0f);
+
+        const int width = (int)svgImage->width;
+        const int height = (int)svgImage->height;
+        // Delete
+        nsvgDelete(svgImage);
+
+        return LoadImageSvgWithSize(fileName, width, height);
+    }
+
+
+    return image;
+}
+
+// Load an image from SVG file data with a custom size
+Image LoadImageSvgWithSize(const char* fileName, int width, int height)
+{
+    Image image = { 0 };
+
+    unsigned int dataSize = 0;
+    unsigned char* fileData = LoadFileData(fileName, &dataSize);
```

Comment:
Be careful with pointer `*`, raylib conventions place it on the right.

Reply:
Reviewed.